packages/manager/modules/hub/src/components/user-panel/payment-mean/template.html

```diff
@@ -0,0 +1,35 @@
+<div class="manager-hub-payment-mean">
+    <button
+        class="d-flex flex-row align-items-center p-2 px-3"
+        data-ng-click="$ctrl.goToPaymentMethods()"
+    >
+        <span class="mr-auto oui-icon oui-icon-credit-card_concept"></span>
+        <oui-skeleton
+            size="s"
+            data-ng-repeat="i in [1,2]"
+            data-ng-if="$ctrl.isLoading"
+        ></oui-skeleton>
+        <div class="m-auto" data-ng-if="!$ctrl.isLoading">
+            <strong
+                class="d-block"
+                data-translate="hub_payment_mean_title"
+            ></strong>
+            <span
+                class="d-block"
+                data-ng-bind="$ctrl.paymentMethod.label"
+            ></span>
+            <span
+                class="oui-status"
+                data-ng-class="{
+                    'oui-status_info': ['CREATED', 'CANCELING', 'CREATING'].includes($ctrl.paymentMethod.status),
```

Comment:
Could this be handled by a class instead ? (to avoid having such logic in the view )

Reply:
what do you mean? a class with methods such has isInfo, isError etc ?

Reply:
Yes something like that :)